services/rds/src/main/java/org/finra/gatekeeper/services/auth/GatekeeperRoleService.java

```diff
@@ -181,4 +225,64 @@ private GatekeeperRdsRole checkGatekeeperRdsRole() {
             return GatekeeperRdsRole.UNAUTHORIZED;
         }
     }
+
+
+
+    public Map<String, Map<String, Map<String, Integer>>> getApprovalPolicy(Map<String, RoleMembership> roleMemberships) {
+        Map<String, Map<String, Map<String, Integer>>> approvalPolicy = new HashMap<>();
+        for(String application : roleMemberships.keySet()) {
+            approvalPolicy.put(application, getApprovalPolicyByApplication(roleMemberships.get(application)));
+        }
+        return approvalPolicy;
+    }
+
+    private Map<String, Map<String, Integer>> getApprovalPolicyByApplication(RoleMembership roleMembership) {
+        gatekeeperRoles = new ArrayList<>();
+        gatekeeperRoles.add("datafix");
+        gatekeeperRoles.add("readonly_confidential");
+        gatekeeperRoles.add("readonly");
+        gatekeeperRoles.add("dba_confidential");
+        gatekeeperRoles.add("dba");
```

Comment:
You should be using the roles from this enum, not hardcoded strings. This can introduce potential issues down the line in the event of roles being added/removed 

https://github.com/FINRAOS/Gatekeeper/blob/master/services/db/src/main/java/org/finra/gatekeeper/rds/model/RoleType.java

I recommend something like this

```suggestion
            Arrays.stream(RoleType.values()).map(RoleType::getDbRole).collect(Collectors.toList());
```